common/rest/request.go

```diff
@@ -51,12 +51,14 @@ import (
 	"net/textproto"
 	"net/url"
 	"strings"
+
+	"github.com/IBM-Cloud/ibm-cloud-cli-sdk/bluemix/models"
 )
 
 const (
-	contentType               = "Content-Type"
-	jsonContentType           = "application/json"
-	formUrlEncodedContentType = "application/x-www-form-urlencoded"
+	ContentType               = "Content-Type"
+	JSONContentType           = "application/json"
+	FormUrlEncodedContentType = "application/x-www-form-urlencoded"
```

Comment:
**Callout**: Exported variables as a result of changing the package name `rest` to `rest_test` in `request_test.go`